source/reference/headings.txt

```diff
@@ -0,0 +1,155 @@
+.. _style-guide-headings:
+
+========
+Headings
+========
+
+.. default-domain:: mongodb
+
+Use over and underlines consisting of a repeating character to specify a heading
+in rST.
+
+Heading Level Markup
+--------------------
+
+The character used for the over and underlines has no effect on the heading
+level, as long as the order used is consistent. However, to maintain consistency
+and readability throughout the documentation, use the following heading level
+structure:
+
+.. list-table::
+   :header-rows: 1
+   :widths: 10 30 10 40 10
+
+   * - Heading Level
+     - Character
+     - Under or Overline
+     - Common Usage
+     - Renders in HTML as
+
+   * - 1
+     - ``=`` (Equals sign)
+     - Both
+     - Page Titles Only
+     - ``<h1>``
+
+   * - 2
+     - ``-`` (Hyphen)
+     - Under
+     - Main Headings
+     - ``<h2>``
+
+   * - 3
+     - ``~`` (Tilde)
+     - Under
+     - Subheadings
+     - ``<h3>``
+
+   * - 4
+     - ````` (Backtick)
+     - Under
+     - 2nd Level Subheadings
+     - ``<h4>``
+
+   * - 5
+     - ``+`` (Plus sign)
+     - Under
+     - 3rd Level Subheadings
+     - ``<h5>``
+
+Heading Markup Rules
+--------------------
+
+Headings must follow these rules to prevent build errors:
+
+- Make the underline at least as long as the text above it.
```

Comment:
Question: Is it actually "at least"? I always thought it had to be an exact match. I've never tried a longer underline (intentionally).

Reply:
I thought the same! But looking into it, it looks like it is "at least". Top and bottom still do have to match though even if they're longer.